node/subsystem-types/src/messages.rs

```diff
@@ -769,8 +769,8 @@ pub enum ApprovalVotingMessage {
 	/// Check if the assignment is valid and can be accepted by our view of the protocol.
 	/// Should not be sent unless the block hash is known.
 	CheckAndImportAssignment(
-		IndirectAssignmentCert,
-		CandidateIndex,
+		IndirectAssignmentCertV2,
+		Vec<CandidateIndex>,
```

Comment:
I do not understand how we know the candidate indices here.

Reply:
We send them over the wire in the message that wraps the certificate.

Reply:
Are we signing that message somehow?

Reply:
no, we don't sign it